test_utilities/bin/dart_test_runner.sh

```diff
@@ -14,6 +14,10 @@ pushd "$1" > /dev/null
 flutter clean
 dart pub get
 
+echo "############# update proto generated code #############"
+protoc --dart_out=. lib/src/model/proto/**/*.proto
```

Comment:
This only runs the formatter on the bot. It also needs something like `git diff --exit-code` to fail if there are local changes

Reply:
Okay, that makes sense now that I look at it. Thanks! Will update.